src/cases/containers/query-management-container/query-management-container.component.html

```diff
@@ -1,7 +1,21 @@
 <exui-app-header></exui-app-header>
-
 <div class="govuk-width-container">
   <main class="govuk-main-wrapper" id="main-content" role="main">
+    <!-- Error message summary -->
+    <div *ngIf="errorMessages.length > 0" class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+      <h2 class="govuk-error-summary__title" id="error-summary-title">
+        {{'There is a problem' | rpxTranslate}}
```

Comment:
@johnbenjamin-hmcts - possible linting failure; the pipeline linter may ask for a space after `{{` and before `}}`

Reply:
@Daniel-Lam - It was a copy-paste from Toolkit to MC. I have now updated the code.